src/igroupadmin.cpp

```diff
@@ -466,14 +466,25 @@ void usageMain() {
         "certain administrative functions.  It can also be used by regular",
         "users to list groups ('lg') and by the admin for all operations.",
         " ",
+        "After making a new group they will themselves manage, a groupadmin's
+        "first action should be to add their own username to the group.",
+        "They will from that point forward be able to add users to, and remove",
+        "them from, the group.  Those operations permit, as well, the transfer",
+        "of responsibility for the group to another groupadmin.",
+        " ",
+        "Currently, a groupadmin may not remove groups or users from the zone."
+        " ",
         "A blank execute line invokes the interactive mode, where it",
         "prompts and executes commands until 'quit' or 'q' is entered.",
         "Single or double quotes can be used to enter items with blanks.",
         "Commands are:",
         " lg [name] (list group info (user member list))",
         " mkuser Name Password (make a user and set the initial password)",
         " atg groupName userName[#Zone] (add to group - add a user to a group)",
+        "   * NOTE: atg is allowed for a groupadmin only if they belong to the group given,",
+        "     and it is currently empty (no members)."
         " rfg groupName userName[#Zone] (remove from group - remove a user from a group)",
+        "   * NOTE: rfg is allowed for a groupadmin only if they belong to the group given.",
```

Comment:
Same comment here as for `atg`: Please consider moving/adding this message to the `rfgMsgs`.

Reply:
(see atgMsgs comment above.)